tools/spatyper/spatyper.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="spatyper" name="Generate spa type identification" version="@VERSION@+galaxy0" profile="21.01">
+    <macros>
+        <token name="@VERSION@">0.3.3</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">spatyper</requirement>
+    </requirements>
+    <command detect_errors="aggressive"> <![CDATA[
+        spaTyper -f '${input1}' -r '${input2}' > '${output1}' 
+    ]]> </command>
+    <inputs>
+        <param type="data" name="input1" format="fasta" label="SPA Fasta file" help="Staphylococcus aureus - staphylococcal protein A fasta file. This will be assigned a type based on its repeat region."/>
+        <param type="data" name="input2" format="fasta" label="Repeats Library Multifasta" help="A multifasta representing a set of known repeat regions found in staphylococcal protein A. This can be downloaded at (http://spa.ridom.de/dynamic/sparepeats.fasta)"/>
+    </inputs>
+    <outputs>
+        <data name="output1" format="txt" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="input1" value="input.fasta"/>
+            <param name="input2" value="sparepeats.fasta"/>
+            <output name="output1">
+                <assert_contents>
+                    <has_text text = "t11215" />
+                </assert_contents>
+            </output>
+        </test>
+    </tests>
+    <help><![CDATA[
+::
```

Comment:
I also think you need a newline below the `::` here.

I tested this using planemo Galaxy (since I didn't know why the help text still wasn't displaying) and now it displays:

![image](https://user-images.githubusercontent.com/200517/226448251-f9dc3732-9192-4402-b1de-98eeda8ddd29.png)